test/plugin/test_zstd_compressor.rb

```diff
@@ -0,0 +1,58 @@
+require "helper"
+require "fluent/plugin/buf_memory"
+begin
+  require "zstandard"
+rescue LoadError
```

Comment:
Don't ignore `LoadError`. Without `zstandard` gem, this test always fails.

Reply:
apologies. this was copied from another test. fixec